src/main/java/jenkins/jmh/JmhBenchmarkState.java

```diff
@@ -0,0 +1,163 @@
+package jenkins.jmh;
+
+import hudson.WebAppMain;
+import hudson.model.DownloadService;
+import hudson.model.Hudson;
+import hudson.model.JDK;
+import hudson.model.RootAction;
+import hudson.model.UpdateSite;
+import hudson.security.ACL;
+import hudson.util.PersistedList;
+import io.vavr.Tuple2;
```

Comment:
Why not https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/tuple/Pair.html ? Apache Commons Collections is widely used in Jenkins, and I would rather stick to it if no practical difference

Reply:
Done!